Source/monster.cpp

```diff
@@ -39,6 +39,19 @@
 
 namespace devilution {
 
+namespace {
+
+void NewMonsterAnim(int i, AnimStruct *anim, Direction md, AnimationDistributionFlags flags = AnimationDistributionFlags::None, int numSkippedFrames = 0, int distributeFramesBeforeFrame = 0)
+{
+	MonsterStruct *monst = &Monsters[i];
+	auto *pCelSprite = &*anim->CelSpritesForDirections[md];
+	monst->AnimInfo.SetNewAnimation(pCelSprite, anim->Frames, anim->Rate, flags, numSkippedFrames, distributeFramesBeforeFrame);
+	monst->_mFlags &= ~(MFLAG_LOCK_ANIMATION | MFLAG_ALLOW_SPECIAL);
+	monst->_mdir = md;
+}
+
+} // namespace

```

Comment:
The defines aren't used, so it's not necessary.
I also wasn't sure where to insert the anonymous namespace. 😕 
It could also be done after all the globals. Perhaps before `InitMonsterTRN`?
But we could also defer this to later when we need the stuff.

Reply:
Yeah I know it is not "necessary" per se, but was just wondering about best practices and consistency. I guess this is more a question to @AJenbo and @glebm here. It's just that I've always seen #defines come at the very top of the file.

Reply:
I looked at player.cpp there the anonymous namespace is directly after devilution namespace.
So I did the same. 😄

Reply:
I would prefer that they be converted in to `constexpr`, in both cases I would have them at the top of the namespace.

Reply:
But I haven't cleaned that file yet :o